### DISCUSSION TITLE
Enable money in mission mode/pvp.

### DISCUSSION THREAD

**Discussion by Crazyjr20** [FakeFishGames/Barotrauma#15274] [Ideas and suggestions]
Currently as it stands, wallets and money as a whole are disabled in mission mode, using the command money in console in these modes returns the following: "No campaign active!"

I understand that there isn't anyway to make money in vanilla's mode, but if its not a huge amount of refactoring work enabling this in the other modes, it could open a lot of doors for modders with no downsides to people who don't interact with it anyway. 
_(E.G personally I would use this to let players earn money from kills in Outpost Deathmatch then spend it in custom vending machines)._

**Comment by AziKarnage** [comment]
Add this to sandbox as well and it will open up tons of possibilities, we have already seen game modes using campaign just to use its economy on maps
